PDE/advection1reverse.m

```diff
@@ -0,0 +1,103 @@
+function pde = advection1reverse
+%1D Test case of the 1D advection equation with
+% the direction of motion reversed from advection1.m.
+% Code designed to test inhomogeneous Dirichlet BC in conjunction with
+% advection1.m
+% Equation to be solved is 
+% df/dt = df/dx + source(x)
```

Comment:
As before, update this to be the actual equation that is being solved

Reply:
The comment has been updated to the actual equation being solved.